experimental/benchmarks/acmeair/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:14.04
+MAINTAINER Gareth Ellis <gareth.ellis@uk.ibm.com>
+RUN apt-get update && apt-get install git openjdk-7-jdk mongodb zip unzip numactl bc wget -y
```

Comment:
This is leaving behind a lot of unnecessary cache files that will be backed into the layer. We should whack the chache after the install. See the [official images](https://github.com/docker-library/buildpack-deps/blob/master/jessie/Dockerfile#L39) for inspiration